src/Http/Routing/src/DefaultLinkGenerator.cs

```diff
@@ -280,13 +297,43 @@ private List<RouteEndpoint> GetEndpoints<TAddress>(TAddress address)
             return null;
         }
 
+        public override RouteValueDictionary ParsePathByAddress<TAddress>(TAddress address, PathString path)
+        {
+            var endpoints = GetEndpoints(address);
+            if (endpoints.Count == 0)
```

Comment:
How/where is GetEndpoints implemented? This was my issue, getting access to the routes/endpoints that used to be in RouteData.Routes

Reply:
Get endpoints is implemented on the base class. 

If you want a list of all of the endpoints in the app, you do do that by grabbing `EndpointDataSource` from DI. If you want the endpoint that matched the current request, use this.  https://github.com/aspnet/AspNetCore/blob/425c196cba530b161b120a57af8f1dd513b96f67/src/Http/Http.Abstractions/src/Routing/EndpointHttpContextExtensions.cs